### ISSUE TITLE
Associated Emails Display -

### ISSUE THREAD

**Issue by servicefly** [servicefly/Education-Features-Bugs#56]
Display "School has associated school emails" that shows both holdlisted and denylisted when an applicant uses a non associated domain like gmail. Additionally, it would be helpful to change the rejection reason from "Allowlisted School Email Not Used" to something like, "Associated School Email Not Used." We as a team are also curious to know what the exact wording is on the current allowlisted email not used rejection reason.
-Here is a case where it works well for allowlisted domain not used: https://edutools.githubapp.com/discount_requests/4473292

-Here is a case where it doesn't show the holdlisted emails associated: https://edutools.githubapp.com/discount_requests/4473297

-Here is a case where a student applied with a holdlisted email and the selected school has allowlisted domains flag is also appearing. If a student applies with an already associated (holdlisted or denylisted) email, I don't think we need to have the additional flag regarding allowlisted emails as they might not have another email. It would be nice to have that flag turned off when a student applies with an associated domain: https://edutools.githubapp.com/discount_requests/4473273

**Comment by divais** [comment]
> -Here is a case where a student applied with a holdlisted email and the selected school has allowlisted domains flag is also appearing. If a student applies with an already associated (holdlisted or denylisted) email, I don't think we need to have the additional flag regarding allowlisted emails as they might not have another email. It would be nice to have that flag turned off when a student applies with an associated domain: https://edutools.githubapp.com/discount_requests/4473273

Does it help knowing that the school has 2 email addresses, one hold listed and another allow listed to help make the decision?

**Comment by ghost** [comment]
Not really. The allowlisted school email not used has become a catch-all for, “we just want you to re apply with a school email.” If we know there’s an email associated to the school and they used a non academic email, like Gmail, then we reject regardless of how many domains and in what state. Unless the higher ups think otherwise, I think a simple red - school has associated domains flag would be best. That could also allow for an automated rejection down the road or be an application pause like requiring photo proof so it gets taken off our plate.
